cmake/multi_image.cmake

```diff
@@ -72,6 +72,11 @@ endfunction()
 function(add_child_image_from_source name sourcedir)
   message("\n=== child image ${name} begin ===")
 
+  # Set ${name}_BOARD based on what BOARD is set to if not already set by parent
+  if (NOT ${name}_BOARD)
+    image_board_selection(${BOARD} ${name}_BOARD)
+  endif()
+
```

Comment:
We already have a mechanism for having the BOARD be set by something else than image_board_selection. Namely cache variables.

This mechanism is used for instance when the name_board is specified through the CMake CLI.

Could/should we have the parent set the board as a cache variable instead?

Reply:
Nice, will double check that it works.

Reply:
My proposal does not work.